specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2019-01-01-preview/cpimTenant.json

```diff
@@ -0,0 +1,917 @@
+        "value": {
+          "description": "List of guest usages resources",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/B2CTenantResource"
+          }
+        }
+      }
+    },
+    "CreateTenantRequestBody": {
+      "type": "object",
+      "description": "The information needed to create the Azure AD B2C tenant and corresponding Azure resource, which is used for billing purposes.",
+      "required": [
+        "location",
+        "sku",
+        "properties"
+      ],
+      "x-ms-azure-resource": true,
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The location in which the resource is hosted and data resides. Refer to [this documentation](https://aka.ms/B2CDataResidency) to see valid data residency locations. Please choose one of 'United States', 'Europe', and 'Asia Pacific'."
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "createTenantProperties": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/CreateTenantProperties"
+            }
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/B2CResourceSKU"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource Tags"
+        }
+      }
+    },
+    "CreateTenantProperties": {
+      "type": "object",
+      "description": "These properties are used to create the Azure AD B2C tenant. These properties are not part of the Azure resource.",
+      "properties": {
+        "displayName": {
+          "description": "The display name of the B2C tenant.",
+          "type": "string"
+        },
+        "countryCode": {
+          "$ref": "#/definitions/CountryCode"
+        }
+      }
+    },
+    "CountryCode": {
+      "type": "string",
+      "description": "Country code of Azure tenant (e.g. 'US'). Refer to [aka.ms/B2CDataResidency](https://aka.ms/B2CDataResidency) to see valid country codes and corresponding data residency locations. If you do not see a country code in an valid data residency location, choose one from the list."
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
+        }
+      },
+      "description": "An error response for a resource management request."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.",
+      "x-ms-parameter-location": "method"
```

Comment:
This is quite peculiar for a subscription id parameter, usually in the generated SDKs, subscriptionID will be defined as a client parameter - not a method parameter. This might bring inconsistency to the customers.